### PR TITLE
konnectivity server now logs to stdout

### PR DESCRIPTION
**Pull request by vpnachev**
**How to categorize this PR?**

/area control-plane networking ops-productivity
/kind enhancement
/priority normal

**What this PR does / why we need it**:
Konnectivity ~agent and~ server logs to stdout. Currently these components are dumping their logs in files in the pod ephemeral filesystem, but the container images comes without shell and basic tools which means that the logs cannot be consumed. Switching to stdout allows plain `kubectl logs <pod-name>` to be used, also the logs from the seed component will be preserved in the Loki instance (cc @Kristian-ZH @vlvasilev new log parser might be needed for this container).

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
The `konnectivity-server` is now logging to the stdout.
```

### CODE REVIEW

**Comment by vpnachev** [comment]
/hold

It turns out that the proxy agent is logging with rate of 700 log messages per second.
There is a upstream improvement PR https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/112 (cc @mvladev ).

**Comment by vpnachev** [comment]
The proxy-server is looking much better - with a single node shoot cluster the log message rate is 3 messages per second.
Maybe I will split the PR change only the server settings.

**Comment by vpnachev** [comment]
/unassign
/needs/review

File: charts/seed-controlplane/charts/kube-apiserver/templates/deployment.yaml [hunk 1]
```diff
@@ -281,9 +281,7 @@ spec:
           - /proxy-server
         args:
         - --uds-name=/etc/srv/kubernetes/konnectivity-server/konnectivity-server.socket
-        - --log-file=/var/log/konnectivity-server/info.log
```
Comment (rfranzke) [inline charts/seed-controlplane/charts/kube-apiserver/templates/deployment.yaml:284]: The `/var/log/konnectivity-server` mount can be removed now right?

Comment (vpnachev) [reply charts/seed-controlplane/charts/kube-apiserver/templates/deployment.yaml:284]: Nice catch, thanks!

**Review by rfranzke — APPROVED** [review]
/lgtm

**Pull request merged**